src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterTest.java

```diff
@@ -103,7 +103,7 @@ public void shouldThrowInvalidHIU() {
     }
 
     @Test
-    void shouldCallGatewayWithHiRequest() {
+    public void shouldCallGatewayWithHiRequest() {
```

Comment:
you can remove public modifier over here

Reply:
not sure. because, in older tests, modifier is public

Reply:
so we are just following the convention

Reply:
yeah.. but it is a code smell

Reply:
removed